### PR TITLE
Docker push implementation

### LINKED ISSUE #167: docker image push command
`docker push [OPTIONS] NAME[:TAG]`

### PR DESCRIPTION
**Pull request by thilakarajk**
Fixes #167 
- Added docker push command

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ List of some daily used commands for tech geeks.
   - [Save docker image as package](commands/docker/docker-save.md)
   - [Tail docker container logs](commands/docker/docker-logs-tail.md)
   - [Tag docker image](commands/docker/docker-tag.md)
+  - [Push docker image to registry](commands/docker/docker-push.md)
```
Comment (arshadkazmi42) [inline README.md]: Can you move this in alphabetical order

File: commands/docker/readme.md [hunk 2]
```diff
@@ -23,3 +23,4 @@ Daily used docker commands
 - [Save docker image as package](docker-save.md)
 - [Tail docker container logs](docker-logs-tail.md)
 - [Tag docker image](docker-tag.md)
+- [Push docker image to registry](docker-push.md)
```
Comment (arshadkazmi42) [inline commands/docker/readme.md]: Can you move this in alphabetical order

File: README.md (hunk 1 above)
Comment (thilakarajk) [reply README.md]: Resolved

File: commands/docker/readme.md (hunk 2 above)
Comment (thilakarajk) [reply commands/docker/readme.md]: Order updated

**Pull request merged**